packages/components/src/hooks/use-tradershub-tracking.ts

```diff
@@ -0,0 +1,113 @@
+import { Analytics } from '@deriv/analytics';
+import { useStore } from '@deriv/stores';
+import { useCallback, useEffect, useMemo } from 'react';
+
+// This hook is used to track the onboarding form in TradersHub
+export const useTradersHubTracking = () => {
+    const { traders_hub, ui, client } = useStore();
```

Comment:
i dont think that its a good idea to put this inside components since it's just for tradershub. i believe the app-store package is the correct place for it.

Reply:
I agree, the fact that the name of hook also indicates that it's related to traders hub only, so I don't think we could reuse it on some other place. Good catch.

I will move this inside `appstore` package

Reply:
hook has been moved to `appstore` package now

Reply:
thanks